### DISCUSSION TITLE
我使用的时候还是有因为堆栈过大而闪退的现象

### DISCUSSION THREAD

**Discussion by liaojiaao** [dnSpyEx/dnSpy#355] [Q&A]
我用旧的dnspy（非dnspyex）都能不闪退，而是报错，我有个类很大，每次打开都会报stack 堆栈空间不足，然后打不开，换了这个dnspyex，直接软件崩溃闪退了

**Comment by ElektroKill** [comment]
Hello, can you please provide the file you are looking at? You can upload it on GitHub or send it to me via email elektrokilldev@protonmail.com
